digiroad2-api-oth/src/test/scala/fi/liikennevirasto/digiroad2/dataimport/MassTransitStopCsvImporterSpec.scala

```diff
@@ -1,18 +1,19 @@
 package fi.liikennevirasto.digiroad2.dataimport
 
 import com.github.tototoshi.csv.{CSVReader, DefaultCSVFormat}
-import fi.liikennevirasto.digiroad2.Digiroad2Context.userProvider
+import fi.liikennevirasto.digiroad2.Digiroad2Context.{roadLinkService, userProvider}
```

Comment:
miksi tää import Digiroad2Context.roadLinkService? 

Näin ei sai tapahtua testi puolella.

Reply:
Paikalliseen testaamiseen, mutta unohtui korvata lopuksi versionhallintaan sopivaksi. Hyvä huomio.